packages/neuron-ui/src/components/SendFromMultisigDialog/index.tsx

```diff
@@ -50,7 +51,9 @@ const SendFromMultisigDialog = ({
     try {
       validateTotalAmount(totalAmount, fee, balance)
     } catch (err) {
-      errorMessageUnderTotal = t(err.message)
+      if (err instanceof AmountNotEnoughException) {
+        errorMessageUnderTotal = t(err.message)
+      }
```

Comment:
Is `else` arm ignored on purpose?

Reply:
I added this if statement because editor tells me `Object is of type 'unknown'.ts(2571)`. And I found the try block at line 52 only calls `validateTotalAmount` which only throws `AmountNotEnoughException`, so I did not include an else arm.

Reply:
Gotcha, but better to add a comment to remind us of this, or it may be treated as a bug in the future.